plugin/src/api/asyncRequests.ts

```diff
@@ -45,30 +45,30 @@ export async function asyncGet(method: string, getterMethod: string): Promise<Re
 }
 
 export async function waitProcess(pid: string): Promise<string> {
-  const response = await fetch(`${apiUrl}/process_status/${pid}`, {
-    method: 'GET',
-    redirect: 'follow',
-    headers: {
-      'Content-Type': 'application/octet-stream'
+  while (true) {
```

Comment:
@taco-paco Curious why we are doing this ?. can it cause infinite loop in some cases ?